src/axes/categoryAxis.ts

```diff
@@ -74,6 +76,80 @@ module Plottable.Axes {
     }
 
     /**
+     * Gets the current alignment. If not set, null will be returned.
+     */
+    public tickTextAlignment(): string;
+    /**
+     * Set alignment of tick labels. Must be one of "left", "right",
+     * or "center", or a falsy value. Setting to a falsy value will
+     * reset tick label alignment to the default alignment.
+     *
+     * Note that alignment is applied before rotation (see "{@link
+     * tickLabelAngle}").
+     * 
+     * @param {String} tickTextAlignment One of "left", "right", "center",
+     * or a falsy value.
+     *
+     * @returns {Category} The calling Category.
+     */
+    public tickTextAlignment(tickTextAlignment: string): this;
```

Comment:
since we're on TS 1.8, we could theoretically use string literal types, but that might break consumers who are using TS 1.7, so we should probably hold off.

cc @JKillian for thoughts on the implications of this